docs/kb/rf-increase.rst

```diff
@@ -9,13 +9,16 @@ How to Safely Increase the Replication Factor
 **Audience: ScyllaDB administrators**
 
 
-Issue
------
+Issues
+------
 
 When a Replication Factor (RF) is increased, using the :ref:`ALTER KEYSPACE <alter-keyspace-statement>` command, the data consistency is effectively dropped
 by the difference of the RF_new value and the RF_old value for all pre-existing data.
 Consistency will only be restored after running a repair.
```

Comment:
The above is not true for tablets, we preserve consistency on RF++. We should fix that (separately). @annastuchlik